csub.py

```diff
@@ -760,22 +753,21 @@ def parsetime(time_str):
 
         # try and resume job with specified name
         (base, arrayid) = checkResume(resume_job_name)
+
         if base:
             # make sure it also works correctly for array jobs
             if arrayid:
                 arrayspec = arrayid
 
+            try:
+                f = open(base, "r")
+                basetxt = f.read()
+                f.close()
+            except Exception, err:
```

Comment:
dont' catch Exception, but the correct thing you're expecting, like OSError or IOError